Content.Server/BodySystem/BodyManagerComponent.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Robust.Shared.BodySystem {
+    public enum BodyPartCompatability { mechanical, biological, universal };	
+	public enum BodyPartType { other, torso, head, arm, hand, leg, foot };	  
+
+	public class BodyManagerComponent : Component {
+		private BodyTemplate _template;
+		private Dictionary<string, BodyPart> _parts;
+
+        public override string Name => "BodyManager";
+
+        public override void ExposeData(ObjectSerializer serializer) {
+            base.ExposeData(serializer);
+            //serializer.DataFieldCached(ref _heatResistance, "HeatResistance", 323);
+        }
+
+        public BodyManagerComponent(BodyTemplate _template){
```

Comment:
You can't control component instantiation so you can't pass parameters to the constructor like this.

You'll want to specify the template via a prototype with `ExposeData` and then load it in `Initialize`.